CONTRIBUTING.md

```diff
@@ -30,15 +30,12 @@ and Python log messages in the debug console under "Python Server".
 
 ## Release
 
-- Bump the version in `package.json` and `pyproject.toml` (use even numbers for stable releases).
-- Bump the `ruff` and `ruff-lsp` versions in `pyproject.toml`.
-- Update the `ruff` version in the README.md
-  - in the Base URLs
-  - in "The extension ships with `ruff==...`"
-- Make sure you have Python 3.7 installed
-- Run `uv venv --python 3.7 && source .venv/bin/activate` to create a Python 3.7 venv and activate it.
-- Run `rm requirements.txt requirements-dev.txt && just lock` to update `ruff` and `ruff-lsp`.
-- Update the Changelog
+- Make sure you have Python 3.7 installed and locatable by uv.
+  (If you're using pyenv, you may need to run `pyenv local 3.7`.)
+- Run `uv run --python=3.7 scripts/release.py`.
+  (Run `uv run --python=3.7 scripts/release.py --help` for information on what this script does,
```

Comment:
Could you add this as a `just` script. E.g. `just release`?

Reply:
I've added this. I haven't documented it as "the recommended way" to create a release, though, as it doesn't look like there's a way to pass the various optional flags (`--help`, `--new-version`, `--new-ruff`, etc.) when you run it as a `just` script. justfiles don't seem to support optional arguments (only arguments with default values, which isn't _quite_ what I want here).